Resources/Prototypes/Body/Organs/arachnid.yml

```diff
@@ -1,42 +1,17 @@
 - type: entity
   id: BaseArachnidOrgan
-  parent: BaseItem
   abstract: true
+  suffix: arachnid
   components:
   - type: Sprite
     sprite: Mobs/Species/Arachnid/organs.rsi
-  - type: Organ
-  - type: Food
-  - type: Extractable
-    grindableSolutionName: organ
-  - type: SolutionContainerManager
-    solutions:
-      organ:
-        maxVol: 10
-        reagents:
-        - ReagentId: Nutriment
-          Quantity: 10
-      food:
-        maxVol: 5
-        reagents:
-        - ReagentId: UncookedAnimalProteins
-          Quantity: 5
-  - type: Tag
-    tags:
-      - Meat
 
 - type: entity
+  parent: [OrganAnimalStomach, BaseArachnidOrgan]
```

Comment:
I don't think we should ever inherit from the animal organs on the roundstart species (and in some other cases, like rat king). There is a pricing difference, some other minor details that we either forget about, or have to manually change back anyway (such as the comparatively big block of redefining the entire SolutionContainerManager just to restore the stomach volume.
It would probably be cleaner and shorter to start from the BaseOrgan<Organ> prototypes and add what few values change compared to that